### PR TITLE
DDF-04746 use shared eslint-config

### PR DESCRIPTION
**Pull request by rymach**
this should resolve issues with IDEs not picking up eslint settings
https://github.com/connexta/ace/issues/14

#### Who is reviewing it? 

@leo-sakh 
@bellcc 
@djblue 
@gjvera 
@Bdthomson 

#### Ask 2 committers to review/merge the PR and tag them here.

@adimka
@AzGoalie

#### How should this be tested?

build
#### What are the relevant tickets?
Fixes: #____

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: ui/package.json [hunk 1]
```diff
@@ -29,7 +28,8 @@
     "react-dom": "16.8.6"
   },
   "devDependencies": {
-    "@connexta/ace": "git://github.com/connexta/ace.git#235d5b3885e1525e2f42bfbd64cd9715d85f498c",
+    "@connexta/ace": "git://github.com/connexta/ace.git#a821ea66f6ecb4404a6a23e9e86a95a0064a113f",
+    "@connexta/eslint-config-connexta": "git://github.com/connexta/eslint-config-connexta.git#3f0caece6210d25c16863d472026fbfa069521a6",
```
Comment (Bdthomson) [inline ui/package.json]: We need another repo for this? Why can't it live in ace?

Comment (rymach) [reply ui/package.json]: afaik eslint expects shared configs to be packaged as a node module with specific naming https://eslint.org/docs/developer-guide/shareable-configs and it seemed easier to make a whole new repo

**Comment by rymach** [comment]
build now

**Comment by rymach** [comment]
@djblue

**Pull request merged**